pkg/graph/schema/types/translation.graphql

```diff
@@ -79,3 +79,111 @@ type TranslationFieldWithOptions {
   options: Map!
 }
 
+"""
+Represents a translation question with no options and a parent
+"""
+type TranslationFieldWithParent {
+  gqlField: String!
+  goField: String!
+  dbField: String!
+  label: String!
+  readonlyLabel: String
+  sublabel: String
+  multiSelectLabel: String
+  isArray: Boolean
+  dataType: TranslationDataType!
+  formType: TranslationFormType!
+  """
+  Is a question a followup to another that doesn't designate it's own readonly question/line
+  """
+  isOtherType: Boolean
+  """
+  Field name for the parent question for fields that represent Other, Please specify, etc.  Used in change history to render parent question for context
+  """
+  otherParentField: String
+  parentRelation: TranslationField!
+}
+
+union ChildTranslation = TranslationFieldWithParent | TranslationFieldWithOptionsAndParent
+
+"""
+Represents a translation question with options and child/children
+"""
+type TranslationFieldWithOptionsAndChildren {
+  gqlField: String!
+  goField: String!
+  dbField: String!
+  label: String!
+  readonlyLabel: String
+  sublabel: String
+  multiSelectLabel: String
+  isArray: Boolean
+  dataType: TranslationDataType!
+  formType: TranslationFormType!
+  """
+  Is a question a followup to another that doesn't designate it's own readonly question/line
+  """
+  isOtherType: Boolean
+  """
+  Field name for the parent question for fields that represent Other, Please specify, etc.  Used in change history to render parent question for context
+  """
+  otherParentField: String
+  options: Map!
+  childRelation: ChildTranslation!
+}
+
+"""
+Represents a translation question with options and parent
+"""
+type TranslationFieldWithOptionsAndParent {
+  gqlField: String!
+  goField: String!
+  dbField: String!
+  label: String! 
+  readonlyLabel: String
+  sublabel: String
+  multiSelectLabel: String
+  isArray: Boolean
+  dataType: TranslationDataType!
+  formType: TranslationFormType!
+  """
+  Is a question a followup to another that doesn't designate it's own readonly question/line
+  """
+  isOtherType: Boolean
+  """
+  Field name for the parent question for fields that represent Other, Please specify, etc.  Used in change history to render parent question for context
+  """
+  otherParentField: String
+  options: Map!
+  parentRelation: TranslationFieldWithOptionsAndChildren!
```

Comment:
Should the parent field be a union type instead? Or if the child is a `TranslationFieldWithOptionsAndParent`, can you safely assume that the parent always is `TranslationFieldWithOptionsAndChildren`?

I had similar questions about the other parent implementations for the related questions.

Reply:
I don't think so, but I'm not sure I fully understand your question.  I was shaping these based on the FE, and we don't really define any parent as a union.  They all have distinct types